playwright-visual.config.ts

```diff
@@ -0,0 +1,77 @@
+import type { PlaywrightTestConfig } from '@sand4rt/experimental-ct-web';
```

Comment:
This looks like a pretty new and fairly niche package, do we know how reliable it's going to be?

Reply:
There's currently an open PR in playwright to absorb this package - https://github.com/microsoft/playwright/issues/20076

This packages functionality / API is identical to the React / Vue component testing libraries offered by Playwright as seen here:
https://github.com/microsoft/playwright/tree/main/packages/playwright-ct-vue
https://github.com/microsoft/playwright/tree/main/packages/playwright-ct-react

I'm pretty confident this will get adopted eventually, it's just due to Lits infancy. Worst case, if this functionality is never natively adopted by Playwright themselves, we can just test in Storybook like we do in Fozzie Components.

Reply:
We're going to temporarily remove this package, and introduce it as part of -https://github.com/justeattakeaway/pie/pull/303

Reply:
Hey guys, i'm the maintainer of the lib and i contributed to Playwright component testing for the other frameworks as well. Glad to see you want to use it! It should work straight forward, but let me know if you have any questions.